OpenRA.Game/Traits/Player/FrozenActorLayer.cs

```diff
@@ -127,6 +129,11 @@ public IEnumerable<IRenderable> Render(WorldRenderer wr)
 
 		public bool HasRenderables { get { return renderables.Any(); } }
 
+		public bool CheckForRemoval(Player owner)
```

Comment:
Minor quibbling: Maybe ```ShouldBeRemoved``` or something? A Boolean method with a name starting "CheckFor..." just sounds weird to me.